packages/types/src/items.ts

```diff
@@ -7,3 +11,84 @@ export type Alterations<T extends Item = Item, K extends keyof T | undefined = u
 	update: (K extends keyof T ? Partial<T> & Pick<T, K> : Partial<T>)[];
 	delete: (K extends keyof T ? T[K] : PrimaryKey)[];
 };
+
+export type ActionEventParams = {
+	event: string | string[];
+	meta: Record<string, any>;
+	context: EventContext;
+};
+
+export type MutationTracker = {
+	trackMutations: (count: number) => void;
+	getCount: () => number;
+};
+
+// TODO: figure out a way to make this generic to be used within @directus/errors
+interface DirectusError<Extensions = void> extends Error {
```

Comment:
This was copied over from `@directus/errors`, as to avoid a circular dependency.

The best option I can think of would be to move the type from `@directus/errors` to `@directus/types` and then make `@directus/types` a dependency of `@directus/errors`.

Reply:
I took a first pass at this, and while there were multiple types that had been named `DirectusError`, I moved the type out and named it `DirectusExtensionsError`. Happy to update it, or change it as needed.